src/openforms/js/components/admin/forms/ButtonContainer.stories.mdx

```diff
@@ -0,0 +1,30 @@
+import {ArgsTable, Canvas, Meta, Story} from '@storybook/addon-docs';
+
+import ButtonContainer from './ButtonContainer';
+
+<Meta title="Django admin/ButtonContainer" component={ButtonContainer} />
+
+# Field
+
+Wrapper for admin add button
```

Comment:
I'm not entirely sure what this component is used for, the name suggests it's a generic container, but it seems to always look like some type of add button:

![image](https://user-images.githubusercontent.com/29249171/216948025-f4f37f79-fed5-4e5b-98c0-2e4da7e404de.png)

Reply:
off the top of my head - this is used when adding a logic action or a form step in a list-like interface :thinking: 

usually used in the `plain` variant I think, because the django button + icon is fugly